Installments/index.jsx

```diff
@@ -13,6 +13,59 @@ const classes = {
   cellHighlight: `${baseClass}__cell__highlight`
 }
 
+const TARGET_FPS = 60
+
+const requestAnimationFrame = (fn) => {
+  if (window.requestAnimationFrame) {
+    window.requestAnimationFrame(fn)
+  } else {
+    setTimeout(fn, 1000 / TARGET_FPS)
+  }
+}
+
+const debounce = (fn) => {
+  let waiting = false
+
+  return function (...args) {
+    if (!waiting) {
+      waiting = true
+
+      requestAnimationFrame(() => {
+        fn(...args)
+        waiting = false
+      })
```

Comment:
I wasn't actually aware that we were using react-motion, since basically nothing in the toolkit is animated at the moment.

The reason I'm messing about with requestAnimationFrame is because we need to reposition the highlight on resize, which is called a million times when you resize the window. Since `calculateHighlightPosition` causes layout due to calling `offsetLeft` etc, it should only be called when we are actually re-rendering.